core/src/test/java/jenkins/xml/XMLUtilsTest.java

```diff
@@ -97,4 +103,16 @@ public void testUpdateByXmlIDoesNotFail() throws Exception {
         assertThat(stringWriter.toString(), containsString("<description>&amp;</description>"));
     }
 
+    /**
+     * Tests getValue() directly. Tests the parse methods too (indirectly - yeah, a purest would have
+     * tests for each).
+     */
+    @Test
+    public void testGetValue() throws XPathExpressionException, SAXException, IOException {
+        URL configUrl = getClass().getResource("/jenkins/xml/config.xml");
+        File configFile = new File(configUrl.getFile());
```

Comment:
No no no...
`URL.getFile()` is not a path that can be uses as a path to a file on the system.  It is the **file part of the URL** which are two different things that overlap only in certain cases.